topics/jenkins/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Kohsuke Kawaguchi
+display_name: Jenkins
+github_url: https://github.com/jenkinsci
```

Comment:
@casz Can you change the github_url to `https://github.com/jenkinsci/jenkins`?

Reply:
Did you see the conversation in the thread?

Reply:
O sorry I missed it. Thanks! I will merge once the build goes green.